mathquill4quill.js

```diff
@@ -308,15 +331,30 @@ window.mathquill4quill = function(dependencies) {
 
           historyDiv = document.createElement("div");
           let container = document.createElement("div");
-          applyHistoryContainerStyles(container);
+          applyHistoryContainerStyles(container, displayDeleteButtonOnHistory);
           let header = document.createElement("p");
           header.innerHTML = "Past formulas (max " + historySize + ")";
           historyDiv.appendChild(header);
 
-          history.forEach(element => {
+          history.forEach((element) => {
```

Comment:
Nit: let's remove this diff to keep the history clean and focused to the minimal required changes for this feature.
```suggestion
          history.forEach(element => {
```